.travis.yml

```diff
@@ -62,7 +64,7 @@ install:
 before_script:
   - mkdir ${PWD}/mongodb-linux-x86_64-${MONGODB}/data
   - ${PWD}/mongodb-linux-x86_64-${MONGODB}/bin/mongod --dbpath ${PWD}/mongodb-linux-x86_64-${MONGODB}/data --logpath ${PWD}/mongodb-linux-x86_64-${MONGODB}/mongodb.log --fork
-  - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then flake8 .; else echo "flake8 only runs on py27"; fi   # Run flake8 for py27
+  - flake8 .
```

Comment:
flake8 will now run under all python versions=

Reply:
Does it really make sense to run `flake8` for all Python versions? Does it do a different set of checks depending on the Python version? If not, we really only need to run it once. That said, we could change the version we run it for, e.g. we could use Python v3.7.

Reply:
Nevermind, I've had a chat with @tsx (my coworker who's leading our py2 -> py3 transitions) and he says that it *does* make sense. When run on py3, flake8 for example detects deprecated exception throwing as well as `unicode` calls, etc.